numpy/polynomial/hermite.py

```diff
@@ -770,8 +770,8 @@ def hermint(c, m=1, k=[], lbnd=0, scl=1, axis=0):
     Raises
     ------
     ValueError
-        If ``m < 0``, ``len(k) > m``, ``np.isscalar(lbnd) == False``, or
-        ``np.isscalar(scl) == False``.
+        If ``m < 0``, ``len(k) > m``, ``np.ndim(lbnd) != 0``, or
+         ``np.ndim(scl) != 0``.
```

Comment:
This works fine for `scl = np.array(0)`, so was previously overly restrictive

Reply:
Yeah, `scl` is not actually checked either.

Reply:
Since I put both `lbnd` and `scl` in, all I can say is, "I don't know nothin' ".